app/src/components/Text.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import * as React from 'react';
```

Comment:
Maybe we should have components as a workspace

Reply:
We should mainly use the `universal-component` for components, but of course, we will need some margarita's specific. But doesn't this raise the complexity? 

I don't have many experiences with workspaces, only when their start using them on Balkan.

Reply:
Why would it raise complexity? The main benefit is I can easily import it like 
`import { Text } from '@kiwicom/margarita-shared';`

rather than

`import Text from '../../../../components/text/Text';`

Reply:
Thanks for the explanation, so yeah it makes sense, to do it as you suggest

Reply:
I will do it in this one, it should not be many changes so the code should be still readable for CR.